selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/ProjectExplorer.java

```diff
@@ -502,7 +502,7 @@ public void waitFolderDefinedTypeOfFolderByPath(String pathToFolder, String type
     String locator =
         "//div[@path='/"
             + pathToFolder
-            + "']//*[local-name() = 'svg' and @id='"
+            + "']/div/*[local-name() = 'svg' and @id='"
```

Comment:
It would be better to use just *id* here as identifier.

Reply:
I agree, but in this case it does not work because we  use @path for define a folder

Reply:
I see.

Reply:
Also i don't have idea what we will be recognize folders with the same type in project tree. We can have many folders with the same type and the same id's. It can be problem and improve this is not trivial task.